src/backend/storeManagers/storeManagerCommon/games.ts

```diff
@@ -33,30 +32,30 @@ export function logFileLocation(appName: string) {
 }
 
 const openNewBrowserGameWindow = async (
-  browserUrl: string
+  browserUrl: string,
+  abortController: AbortController
 ): Promise<boolean> => {
+  const mainUrlName = browserUrl.split('://')[1].split('/')[0]
```

Comment:
How about using `URL()` constructor here instead of this nasty splitting

Reply:
Cool, will check this out.